cloudinit/config/cc_apt_configure.py

```diff
@@ -427,9 +427,11 @@ def mirrorurl_to_apt_fileprefix(mirror):
     return string
 
 
-def rename_apt_lists(new_mirrors, target=None):
+def rename_apt_lists(new_mirrors, target=None, arch=None):
```

Comment:
Best I can tell, this is only called from line 312 (that's also modified in this PR) and tests. Would it be simpler to make all the parameters positional, so we don't have to handle the `arch=None` case (which will never happen in practice)?

Reply:
Sure.  There are a few other methods in that handler that also just take arch as positional parameter.